lang/po/new/cs.po

```diff
@@ -1143,7 +1140,7 @@ msgstr "Reset XYZ kalibr."
 # MSG_RESET c=14
 #: messages.c:85
 msgid "Reset"
-msgstr ""
+msgstr "Reset"

```

Comment:
In case that the translation is the same as the source text, please leave it empty to save some space.